core/src/main.jl

```diff
@@ -14,6 +14,9 @@ function run(config::Config)::Model
     return model
 end
 
+main(ARGS::Vector{String})::Cint = main(only(ARGS))
+main()::Cint = main(ARGS)
```

Comment:
This doesn't seem part of this PR?

Reply:
Not really, but it was broken without, I needed it to run `pixi run ribasim-core ribasim.toml`. These lines were there before.